tools/csv-merger/csv-merger.go

```diff
@@ -589,6 +590,8 @@ The NooBaa operator deploys and manages the [NooBaa][2] Multi-Cloud Gateway on O
 	// Used by UI to validate user uploaded metdata
 	// Metadata is used to connect to an external cluster
 	ocsCSV.Annotations["external.features.ocs.openshift.io/validation"] = `{"secrets":["rook-ceph-operator-creds", "rook-csi-rbd-node", "rook-csi-rbd-provisioner", "rook-csi-cephfs-node", "rook-csi-cephfs-provisioner"], "configMaps": ["rook-ceph-mon-endpoints", "rook-ceph-mon"], "storageClasses": ["ceph-rgw", "ceph-rbd", "cephfs"]}`
+	// Injecting the RHCS exporter script present in Rook CSV
+	ocsCSV.Annotations["external.features.ocs.openshift.io/script"] = rookCSV.GetAnnotations()["externalClusterScript"]
```

Comment:
The change makes sense. However, we should make sure not to back ourselves into a corner and leave room for potentially more scripts in the future.

```suggestion
	ocsCSV.Annotations["external.features.ocs.openshift.io/export-script"] = rookCSV.GetAnnotations()["externalClusterScript"]
```

Reply:
ack. making similar changes in https://github.com/openshift/console/pull/5580 as well.